### PR TITLE
[TFLite] Add extra operators and fixes.

### PR DESCRIPTION
**Pull request by mciprian13**
**Summary**
- Added logic for shape inference for extra operators.
- Some fixes for the SAME/VALID padding computation.
- Try to import ReduceMean as AvgPool if equivalent.

**Test Plan**
Added unit tests in TFLiteImportTest.

### CODE REVIEW

**Comment by mciprian13** [comment]
Ping.

**Review by khabinov — COMMENTED** [review]
LGTM

**Comment by mciprian13** [comment]
@khabinov Thanks for reviewing this. If all good let's have this landed. Thanks!

File: lib/Graph/Nodes.cpp [hunk 1]
```diff
@@ -2580,7 +2580,7 @@ bool MFCCNode::verify() const {
   isValid &= expectCompareTrue(
       "Number of coefficients should be smaller than the filter bank count",
```
Comment (khabinov) [inline lib/Graph/Nodes.cpp:2581]: Actually just realized that the comment needs to be updated as well.

Comment (mciprian13) [reply lib/Graph/Nodes.cpp]: Done

**Comment by mciprian13** [comment]
@khabinov Is there a problem with this PR? Can we have this landed? Thanks!

**Comment by khabinov** [comment]
Hi @mciprian13! Took some time to resolve CI issues with tests, we're currently re-running them.

**Pull request closed without merging**